mobile/src/pages/Register.tsx

```diff
@@ -35,14 +56,18 @@ export default function Register({ navigation }) {
               placeholderTextColor="#000000"
               onChangeText={setEmail}
             />
+
           </View>
+          <Text className="text-red-500 mx-auto font-extrabold">
+            {emailRegex() ? "" : "Please enter a valid email"}
```

Comment:
This can just be `email.includes("@") ? "" : "Please enter a valid email."` once you have email state